src/ext/GeometryUtil.js

```diff
@@ -55,55 +82,67 @@ L.GeometryUtil = L.extend(L.GeometryUtil || {}, {
 	// Converts a metric distance to one of [ feet, nauticalMile, metric or yards ] string
 	//
 	// @alternative
-	// @method readableDistance(distance, isMetric, useFeet, isNauticalMile): string
+	// @method readableDistance(distance, isMetric, useFeet, isNauticalMile, precision): string
 	// Converts metric distance to distance string.
-	readableDistance: function (distance, isMetric, isFeet, isNauticalMile) {
+	// The value will be rounded as defined by the precision option object.
+	readableDistance: function (distance, isMetric, isFeet, isNauticalMile, precision) {
 		var distanceStr,
-			units;
+			units,
+			precision = L.Util.extend({}, defaultPrecision, precision);
 
-		if (typeof isMetric == "string") {
-			units = isMetric;
+		if (isMetric) {
+			units = typeof isMetric == 'string' ? isMetric : 'metric';
```

Comment:
@boris-arkenaar I don't think so, according to this:
```javascript
if (typeof isMetric == "string") {
-			units = isMetric;
+		if (isMetric) {
+			units = typeof isMetric == 'string' ? isMetric : 'metric';
```
typeof isMetric still accepts a string. If it's a boolean and `feet` is not specified, distances will still be measured in metres. Then again, i'm not that good with PR's and it's possible I am looking at the wrong one.